assert/utils.go

```diff
@@ -12,6 +12,8 @@ import (
 	"unicode/utf8"
 )
 
+const maxSimilarLen = 20
```

Comment:
Thanks for doing the change, bit let me challenge you

Please add a comment about this magical number. What is it? Why 20 was chosen?

It's OK to say it's arbitrary or empirical, but it should be clear to the people reading the code